auslib/web/public/releases.py

```diff
@@ -0,0 +1,17 @@
+from auslib.web.admin.views.releases import (
+    ReleasesAPIView, SingleReleaseView, ReleaseHistoryView)
```

Comment:
Unfortunately, this isn't a great idea. The admin and public app are siblings, so they should not be importing each others code. I think most of the endpoints are small enough that copying/reimplementing would be OK, but you're also welcome to create a base or helpers methods that implement each one, and use that for both implementations.